dotcom-rendering/src/components/ManyNewsletterSignUp.importable.tsx

```diff
@@ -212,16 +227,20 @@ export const ManyNewsletterSignUp = () => {
 			const responseText = response
 				? await response.text()
 				: '[fetch failure - no response]';
-			reportTrackingEvent('ManyNewsletterSignUp', 'failure-response', {
-				newsletterIds: newslettersToSignUpFor,
-				responseText,
-			});
+			void reportTrackingEvent(
+				'ManyNewsletterSignUp',
+				'failure-response',
+				{
+					idNumbers,
+					responseText: responseText.substring(0, 100),
```

Comment:
done - its a very crude way to avoid including a whole HTML error page in the event details - if the backend doesn't handle the error state with a message, the response text will be the error page. It seemed like overkill to try and parse the text and determine what it signifies.